djangosaml2/views.py

```diff
@@ -250,14 +250,22 @@ def get(self, request, *args, **kwargs):
 
         # SSO options
         sign_requests = getattr(conf, '_sp_authn_requests_signed', False)
-        if sign_requests:
-            sso_kwargs["sigalg"] = getattr(conf, '_sp_signing_algorithm',
-                                           saml2.xmldsig.SIG_RSA_SHA256
-            )
-            sso_kwargs["digest_alg"] = getattr(conf,
-                                               '_sp_digest_algorithm',
-                                               saml2.xmldsig.DIGEST_SHA256
+        sso_kwargs['requested_authn_context'] = getattr(
+                conf,
+                '_sp_requested_authn_context',
+                None
             )
+        csc = settings.SAML_CONFIG.get(
```

Comment:
please don't use, even/if possibile, django static settings but `getattr(conf, ...` instead.

Reply:
We have a commit for that, please update your branch